content/posts/blog/2021-01-20.md

```diff
@@ -0,0 +1,55 @@
+---
+title: It is not a choice of the messenger
```

Comment:
I'm not actually sure what this title means, but I don't think it translated to English very well. What is a messenger in this context? An individual client? Maybe "Choose networks, not apps" or something to that effect?

Reply:
I think about 'chat messenger' now?

Reply:
I don't think "chat messenger" is a thing either, and I'm not sure what that means. Do you mean an instant messaging app?

Reply:
> It is not a choice of the instant messaging app